CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Updated the toolchain version used by `ink_linting` - [#1616](https://github.com/paritytech/cargo-contract/pull/1616)
```

Comment:
Please add a "Removed" section. See [this for reference](https://github.com/use-ink/cargo-contract/blob/f99fc6cbea000ea5601e35888bf2cd9a8fb112f4/CHANGELOG.md#removed-1) for the removal of v11 metadata.